src/services/SelfTest.ts

```diff
@@ -59,12 +73,14 @@ export default class SelfTest {
           ws.on('message', (message: string) => {
             let finalMessage = JSON.parse(message)
             if (finalMessage.websocketEnabled) {
+              this.config.wsTestSuccess = true
               resolve()
             }
           })
 
           ws.on('error', (err: any) => {
             log.error('Error on connection for websockets', err)
+            this.config.wsTestSuccess = false
```

Comment:
rather than putting two variables in config that say whether the self test succeeded at different points, could we have two variables in the `SelfTest` class and then assign `this.config.testSuccess` when they're both satisfied

Reply:
The general idea sounds good, but since the process just exits if any test fails I'll just have it set the config value to false anytime it exits, and true if it makes it to the end without maintaining any class variables.